specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -6177,6 +6398,152 @@
+          "items": {
+            "type": "string"
+          },
+          "description": "List of allowed verbs"
+        },
+        "apiGroups": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "List of allowed apiGroups"
+        },
+        "resources": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "List of allowed resources"
+        },
+        "resourceNames": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "List of allowed names"
+        },
+        "nonResourceURLs": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "List of allowed nonResourceURLs"
+        }
+      }
+    },
+    "TrustedAccessRole": {
+      "type": "object",
+      "description": "Trusted access role definition",
+      "properties": {
+        "sourceResourceType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type of Azure resource"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of role, name is unique under a source resource type"
+        },
+        "rules": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/TrustedAccessRoleRule"
+          },
+          "x-ms-identifiers": [],
+          "description": "List of rules for the role"
+        }
+      }
+    },
+    "TrustedAccessRoleListResult": {
+      "type": "object",
+      "description": "List of trusted access roles",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/TrustedAccessRole"
+          },
+          "x-ms-identifiers": [
+            "sourceResourceType",
+            "name"
+          ],
+          "description": "Role list"
+        }
+      }
+    },
+    "TrustedAccessRoleBindingProperties": {
+      "type": "object",
+      "description": "Properties for trusted access role binding",
+      "required": [
+        "sourceResourceId",
+        "roles"
+      ],
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current provisioning state of trusted access role binding."
+        },
+        "sourceResourceId": {
+          "type": "string",
+          "description": "The resource id that allowed for trusted access."
+        },
+        "roles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of roles to bind."
```

Comment:
IMO we should give an example in the docs as well as to what one of these looks like. Maybe say: `for example: Microsoft.MachineLearningServices/workspaces/reader`

Reply:
sure updated.